core/include/tangram/map.h

```diff
@@ -99,17 +98,15 @@ class Map {
     // Any pending scene update will be cleared
     void loadSceneAsync(const char* _scenePath,
                         bool _useScenePosition = false,
-                        MapReady _onMapReady = nullptr,
-                        void *_onMapReadyUserData = nullptr,
-                        const std::vector<SceneUpdate>& sceneUpdates = {},
-                        SceneUpdateErrorCallback _onSceneUpdateError = nullptr);
+                        SceneReadyCallback _onSceneReady = nullptr,
```

Comment:
Note: When these changes get merged and released we have to make sure to update tizen plugin with these updates.